lib/still/compiler/view_helpers/responsive_image.ex

```diff
@@ -12,14 +12,15 @@ defmodule Still.Compiler.ViewHelpers.ResponsiveImage do
   @doc """
   Returns an image tag with the `src` and `srcset`.
 
-  If `:image_opts` is set in `opts`, it will be passed on to #{Preprocessor.Image}.
+  If `:sizes` or `:transformations` are present in `opts`, they will be passed
+  to #{Preprocessor.Image}.
```

Comment:
```suggestion
  to `Still.Preprocessor.Image`.
```

This allows the docs to link to the correct page.

Reply:
I saw it written like that somewhere, but yeah, it doesn't seem to link it.